### PR TITLE
Support multiple css paths

### PR DESCRIPTION
**Pull request by randallreedjr**
## Rationale
I am currently using critical-path-css-rails to generate the critical CSS for a react on rails project using webpack to package our assets. We are using the ExtractTextPlugin to separate the CSS for each page to minimize load. We successfully deployed critical-path-css-rails for one of our pages, but cannot easily apply it to additional pages without a way to handle multiple CSS paths.

## Implementation
This is the simplest implementation that occurred to me. It adds a new key, `css _paths`, which is mutually exclusive from the existing key `css_path`. Like `routes`, `css_paths` can be a list of css files that correspond to the routes; we assume they are in the same respective order. Therefore, both arrays should be the same length.

This change is backwards compatible, so existing users should not experience a breaking change.

The PR also updates the README to address #31.

## Tasks
- [x] Add new key `css_paths` to specify separate css path per route
- [x] Add error checking to ensure css_paths is used correctly
- [x] Clarify error messages
- [x] Update Configuration to add new key to config object
- [x] Update CssFetcher to pass separate css_path to each Penthouse invocation
- [x] Update generator to add css_paths key
- [x] Add text to test page that is always visible
- [x] Bump version
- [x] Squash before merging

### CODE REVIEW

**Comment by michael-misshore** [comment]
@randallreedjr Thanks for the PR! I've been taking a look at this, and I have some feedback. I'll try to post all my feedback by this weekend. One thing that I'd like to see done is the new `css_paths` config added to the Usage section in the Readme, along with the other config options. That would be a huge help!

I have a few edit suggestions for the tests (which we can always refactor later so not terribly critical), `css_fetcher.rb`, and `config_loader.rb`. I'll try to post them as quickly as possible.

**Comment by randallreedjr** [comment]
Cool, thanks @michael-misshore! I really appreciate your responsiveness. I'll get cracking on your feedback as soon as I can.

**Comment by dm-randy** [comment]
@michael-misshore Friendly reminder for feedback :)

**Comment by michael-misshore** [comment]
@dm-randy Thanks and sorry!. @randallreedjr Your changes gave me some ideas for some refactors. I started working on this before the holiday, and I'll try to have something that can be reviewed and merged into your branch sometime soon.

**Comment by randallreedjr** [comment]
Oops, both of those accounts are me

**Comment by randallreedjr** [comment]
@michael-misshore I just pushed an update with an addition to the README. Any feedback from your end?

**Comment by michael-misshore** [comment]
@randallreedjr README changes look good. Did you see the review I started regarding `config_loader_spec.rb`? I'd like to hear your feedback (and any changes you make) on it.

**Comment by randallreedjr** [comment]
@michael-misshore Hm, not sure what's going on, but I don't see any reviews

File: spec/lib/critical_path_css/rails/config_loader_spec.rb [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+RSpec.describe 'ConfigLoader' do
+  let(:subject) { CriticalPathCss::Rails::ConfigLoader.new }
+  describe '#load' do
+    before do
+      allow(File).to receive(:read).and_return(config_file)
+    end
+
+    context 'when single css_path is specified' do
+      let(:config_file) {
```
[the remark this answers is not in the document]
Comment (michael-misshore) [reply spec/lib/critical_path_css/rails/config_loader_spec.rb]: @randallreedjr Can you see my above message?

[the remark this answers is not in the document]
Comment (randallreedjr) [reply spec/lib/critical_path_css/rails/config_loader_spec.rb]: @michael-misshore Yep, will do!

**Pull request merged**